### ISSUE TITLE
Doesn't work on Ubuntu 14.10

### ISSUE THREAD

**Issue by rubo77** [fogleman/Ricochet#5]
```
python main.py
Traceback (most recent call last):
  File "main.py", line 4, in <module>
    import ricochet
  File "/tmp/Ricochet/ricochet.py", line 17, in <module>
    dll = CDLL('_ricochet')
  File "/usr/lib/python2.7/ctypes/__init__.py", line 365, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: _ricochet: cannot open shared object file: No such file or directory
```

**Comment by fogleman** [comment]
Did you run `./build_ricochet`?

**Comment by rubo77** [comment]
sure

**Comment by rubo77** [comment]
Sure, But I use Linux Ubuntu 14.10 (32 bit)

maybe this only works in python on windows and there would be a solution using wine?

**Comment by fogleman** [comment]
You might have to rename it to _ricochet.so and possibly change that line to:

    dll = CDLL('_ricochet.so')

**Comment by fogleman** [comment]
You might also have to compile it with `-fPIC` but usually it will tell you that.

**Comment by rubo77** [comment]
how? I tried 
  
    export CXXFLAGS="$CXXFLAGS -fPIC"
    ./build_ricochet 

and I changed in `build_ricochet` the line to _ricochet.so
and in ricochet.py that line too

**Comment by fogleman** [comment]
You can just add -fPIC to the gcc commands in build_ricochet.

**Comment by rubo77** [comment]
I think I tried that already.

Can you post a complete solution?

Maybe, you can download a Ubuntu live CD and try it out?

**Comment by raistlinmaestre** [comment]
Same problem here. I tried compiling with -fPIC and ranaming, but I still get the same error. I'm running Debian, but I supose the problem will be the same in any other Linux distribution.

**Comment by B3n3** [comment]
Hey guys!

I wrote a patch for that problem. 

@fogleman If you want to merge it into master, please contact me via mail - I'd like to be a contributor for that commit :)

Regards, Benedikt


[0001-Fix-for-Ubuntu-14.10.patch.txt](https://github.com/fogleman/Ricochet/files/73090/0001-Fix-for-Ubuntu-14.10.patch.txt)

**Comment by rubo77** [comment]
@B3n3 you should just fork this repository and commit your changes in your branch, then you can start a l"Pull request" with the green buttons.

**Comment by B3n3** [comment]
@rubo77 Thanks :+1: Done.

**Comment by fogleman** [comment]
Fixed by #6
